### PR TITLE
cancelling multi-process  

### PR DESCRIPTION
**Pull request by EvZissel**
envs.py - cancelling the multi-process execution and reward normalization (turns out it preforms better without reward normalization)

main.py - extract the evaluation environments from the evaluate function to create the environments only once (for computational  efficiency)

### CODE REVIEW

File: a2c_ppo_acktr/envs.py [hunk 1]
```diff
@@ -95,16 +95,16 @@ def make_vec_envs(env_name,
...
-        envs = SubprocVecEnv(envs)
-    else:
-        envs = DummyVecEnv(envs)
-
-    if len(envs.observation_space.shape) == 1:
-        if gamma is None:
-            envs = VecNormalize(envs, norm_reward=False)
-        else:
-            envs = VecNormalize(envs, gamma=gamma)
+    # if len(envs) > 1:
```
Comment (avivt) [inline a2c_ppo_acktr/envs.py]: Why did you remove the normalization?

Comment (EvZissel) [reply a2c_ppo_acktr/envs.py]: You're right - the normalization should be here. I thought at the beginning that for this experiment we don't need it, but for the recurrent experiment the normalization improves performance. It needs to be there.
